polars/polars-lazy/src/frame/json.rs

```diff
@@ -0,0 +1,120 @@
+use polars_core::prelude::*;
```

Comment:
Can we move this file to `polars/polars-lazy/src/physical_plan/executors/scan/json.rs`? That's where all `scan` implementations reside.

Reply:
oh, i was basing it off of the csv scan impl in [polars-lazy/src/frame.csv.rs](https://github.com/universalmind303/polars/blob/py-scan-json/polars/polars-lazy/src/frame/csv.rs). 

Since it already leverages the `AnonymousScan` executor in `physical_plan/executors`, does it need it's own exec?

Reply:
No, I mean that we just implement `AnonymousScan` in `polars/polars-lazy/src/physical_plan/executors/scan/json.rs`.

Reply:
i think i understand. I should move the `impl AnonymousScan for LazyJsonReader` into `executors/scan/json.rs`?

Reply:
Yep. :)